framework/functions.php

```diff
@@ -257,14 +257,18 @@ function fw_menu_output ( $menu, $level, $type, $classes ) {
 	echo '">';
 
 	foreach ( $menu as $item ) {
-	
+
 		echo '<li class="';
 
-		if (
-			isset ( $GLOBALS['vars']['current_url'] ) &&
-			$GLOBALS['vars']['current_url'] == $item['url']
-		) {
-			echo 'current-nav-item ';
+		if ( isset ( $GLOBALS['vars']['current_url'] ) ) {
+			$query_position = strpos( $GLOBALS['vars']['current_url'], '?' );
+			$base_current_url = $query_position ?
+				substr( $GLOBALS['vars']['current_url'], 0, $query_position ) :
+				$GLOBALS['vars']['current_url'];
+
+			if ( $base_current_url == $item['url'] ) {
+				echo 'current-nav-item ';
```

Comment:
I don't see this `current-nav-item` class defined in any CSS file.

It's ok to leave it here if we don't intend to use it now, in case we want to use it one day. But just a check if it was forgotten.

Reply:
It's not in use right now, and not part of the requirements. We can add some different styling if wanted.